### PR TITLE
Updating branding for Casper Node

### PR DESCRIPTION
**Pull request by ACStone-MTS**
This PR includes all changes made by the [other branding PR](https://github.com/casper-network/casper-node/pull/3837) that appears abandoned.

Further, I've updated based on [Rafal's PR](https://github.com/0xE69/casper-node/pull/1) and Medha's review of the prior PR.

Related documentation tickets:
[Update branding and company name for node 1.6 #989](https://github.com/casper-network/docs/issues/989)

### CODE REVIEW

**Review by RBYGLLS — APPROVED** [review]
@MParlikar FYI

File: node/src/components/rpc_server/rpcs/docs.rs [hunk 1]
```diff
@@ -37,11 +37,11 @@ const DEFINITIONS_PATH: &str = "#/components/schemas/";
 // As per https://spec.open-rpc.org/#service-discovery-method.
 pub(crate) static OPEN_RPC_SCHEMA: Lazy<OpenRpcSchema> = Lazy::new(|| {
     let contact = OpenRpcContactField {
-        name: "CasperLabs".to_string(),
+        name: "Casper Labs".to_string(),
         url: "https://casperlabs.io".to_string(),
     };
     let license = OpenRpcLicenseField {
-        name: "CasperLabs Open Source License Version 1.0".to_string(),
+        name: "Casper Open Source License Version 1.0".to_string(),
```
Comment (MParlikar) [inline node/src/components/rpc_server/rpcs/docs.rs]: I think this needs to be Apache 2.0 - not COSL.

**Review by MParlikar — CHANGES_REQUESTED** [review]

File: node/src/components/rpc_server/rpcs/docs.rs (hunk 1 above)
Comment (ACStone-MTS) [reply node/src/components/rpc_server/rpcs/docs.rs]: @MParlikar Updated to the Apache license.

**Review by MParlikar — APPROVED** [review]

**Comment by rafal-ch** [comment]
@ACStoneCL @MParlikar 
PR should be good to go now

**Pull request merged**